### PR TITLE
[DNM] raft_cmdpb: transfer leader admin command

### PR DESCRIPTION
**Pull request by tiancaiamao**

### CODE REVIEW

File: proto/raft_cmdpb.proto [hunk 1]
```diff
@@ -109,25 +109,35 @@ message CompactLogResponse {
     
 }
 
+message TransferLeaderRequest {
+    optional metapb.Peer peer =	1;
```
Comment (siddontang) [inline proto/raft_cmdpb.proto]: "peer = 1"

Comment (tiancaiamao) [inline proto/raft_cmdpb.proto]: it seems a bug of github...
open the 'view' button and view the whole file, you'll see there is actually a space here.

**Comment by siddontang** [comment]
LGTM

**Comment by tiancaiamao** [comment]
PTAL @ngaut @BusyJay

**Comment by ngaut** [comment]
LGTM

**Pull request merged**